StandardOutput.txt

```diff
@@ -0,0 +1 @@
+10.0.100-preview.6.25315.102
```

Comment:
Get rid of this file and also of the StandardError.txt


If not present, add them to gitignore perharps?